types/stripe/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for stripe 6.25
+// Type definitions for stripe 6.26
```

Comment:
Could you kindly remove me as a contributor in this PR?
Getting waaay too many emails from this.